docs/reference/concepts/logging.md

```diff
@@ -1,73 +1,147 @@
 
-The SpatialOS GDK for Unity uses a custom [`ILogDispatcher`]({{urlRoot}}/api/core/i-log-dispatcher) interface instead of `UnityEngine.Debug`, which gives more flexibility to handle logs separately in different workers and gives more context when handling the logs. There are two provided implementations of this interface:
+There are two implementations of this interface in the Core package:
 
-*  [`LoggingDispatcher`]({{urlRoot}}/api/core/logging-dispatcher), which simply logs to the Unity console
-*  [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher), which logs to the Unity console and sends it to the SpatialOS Console
-    * Note: By default, messages with the log level `LogType.Log` are not sent to SpatialOS. This can be changed by instantiating the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) with a different `minimumLogLevel` parameter.
+*  The [`LoggingDispatcher`]({{urlRoot}}/api/core/logging-dispatcher), which logs to the Unity console.
+*  The [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher), which logs to the Unity console and sends it to the SpatialOS Console.
 
-All workers use the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) by default in the Playground. If you want to use the [`LoggingDispatcher`]({{urlRoot}}/api/core/logging-dispatcher), see the last step of [Creating and using your own dispatcher]({{urlRoot}}/reference/concepts/logging#creating-and-using-your-own-dispatcher).
+> All workers in the FPS Starter Project and the Blank Project use the [`ForwardingDispatcher`]({{urlRoot}}/api/core/forwarding-dispatcher) by default.
 
-## Using the ILogDispatcher
+## Setting up a log dispatcher
 
-You can access the dispatcher through the [`WorkerSystem`]({{urlRoot}}/api/core/worker-system). The dispatcher provides a single `HandleLog` function, which takes two arguments:
+When you create a worker using the [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector), you pass in an `ILogDispatcher` instance. This associates the `ILogDispatcher` instance with that worker.
 
-* `LogType` (e.g. `UnityEngine.LogType.Error`)
-* `LogEvent`, which stores the message and other context variables in the Data dictionary
+```csharp
+private async void Start()
+{
+    var builder = new SpatialOSConnectionHandlerBuilder()
+        .SetConnectionParameters(CreateConnectionParameters(WorkerUtils.UnityClient))
+        .SetConnectionFlow(new ReceptionistFlow(CreateNewWorkerId(WorkerUtils.UnityClient)));
+
+    // Associate a new ForwardingDispatcher instance with this worker.
+    await Connect(builder, new ForwardingDispatcher()).ConfigureAwait(false);
+}
+```
 
-There are two log context variables:
+This instance handles all logging from the Core and Feature Modules for that worker and is available for you to use through the methods described below
```

Comment:
```suggestion
This instance handles all logging from the Core and Feature Modules for that worker and is available for you to use through the methods described below:
```

Reply:
I actually think this should be a full stop rather than a colon

Reply:
```suggestion
This instance handles all logging from the Core and Feature Modules for that worker and is available for you to use through the methods described below.
```